### DISCUSSION TITLE
v0.0.7 is live 🚀

### DISCUSSION THREAD

**Discussion by AntonOsika** [AntonOsika/gpt-engineer#400] [Announcements]
## What's Changed
* edit on some typing by @abdoafage in https://github.com/AntonOsika/gpt-engineer/pull/341
* Extend tests for DB class by @Snoup97 in https://github.com/AntonOsika/gpt-engineer/pull/128
* Refactoring and suggestions by @k1lgor in https://github.com/AntonOsika/gpt-engineer/pull/334
* refactor: Refactor loop for improved efficiency and readability by @kbshal in https://github.com/AntonOsika/gpt-engineer/pull/286
* fix typos by @Snoup97 in https://github.com/AntonOsika/gpt-engineer/pull/359
* First step in making gpt-engineer learn. Rename main_prompt -> prompt by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/381
* Allow model string to be changed (Fixes #328) by @Explosion-Scratch in https://github.com/AntonOsika/gpt-engineer/pull/329
* Make sure benchmark runs and be more strict about collecting feedback by @AntonOsika in https://github.com/AntonOsika/gpt-engineer/pull/397

## New Contributors
* @abdoafage made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/341
* @k1lgor made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/334
* @kbshal made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/286
* @Explosion-Scratch made their first contribution in https://github.com/AntonOsika/gpt-engineer/pull/329

**Full Changelog**: https://github.com/AntonOsika/gpt-engineer/compare/v0.0.6...v0.0.7

<hr /><em>This discussion was created from the release <a href='https://github.com/AntonOsika/gpt-engineer/releases/tag/v0.0.7'>v0.0.7 is live 🚀</a>.</em>

**Comment by Atsushi888** [comment]
Hello.
I found out about GTP-engineer today and had a gut feeling that it is a very exciting programme and I would love to install it on my iPad.

I knew that the program is for Windows, but I knew it was impossible and tried to install it,

"<class 'ModuleNotFoundError'>: No module named 'urlparse'."

error message appeared.


I know it's impossible, as the programme is still incomplete, but if you don't mind, I would like to be able to use GTP-engineer on my iPad.

Thank you very much.
